tools/machine-setup/README.md

```diff
@@ -144,6 +144,8 @@ iTerm2
 - git config --global rebase.autoStash true
 - git config --global push.autoSetupRemote true
 - git config --global core.editor "code --wait"
+- git config --global alias.pushf 'push --force-with-lease --force-if-includes'
```

Comment:
fyi in case you don't know, oh-my-zsh has a built in git modulo that adds a bunch off git aliases already like `gpf` for this line you added

Reply:
Good to know. I originally added them to my .zshrc directly, but I think it may be better to transition them from `git pull origin main` to `git pullo` first and then transition them to like `gpo` later.

There's a bit too many commands in the plugin that I would prefer not adding if I don't use them.